### PR TITLE
Update latest kueue v0.9.0

### PR DESCRIPTION
**Pull request by mimowo**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Part of #3302 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: charts/kueue/values.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ controllerManager:
       pullPolicy: IfNotPresent
   manager:
     image:
-      repository: us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue
+      repository: registry.k8s.io/kueue/kueue
```
Comment (tenzen-y) [inline charts/kueue/values.yaml:26]: what is this replacement reason?

Comment (mimowo) [reply charts/kueue/values.yaml:26]: I'm not sure. I will check in the next 2-3h

Comment (dgrove-oss) [reply charts/kueue/values.yaml:26]: My guess would be that for a release, the install should not pull from staging-images.

Comment (tenzen-y) [reply charts/kueue/values.yaml:26]: > My guess would be that for a release, the install should not pull from staging-images.

This is for main branch, that indicates development image, IMO.

Comment (mimowo) [reply charts/kueue/values.yaml]: I suspect this was some non-committed change from previous steps, but not sure. I repeated the steps and it didn't generate. PTAL.

Comment (tenzen-y) [reply charts/kueue/values.yaml]: /lgtm

**Pull request merged**